test/flake.nix

```diff
@@ -21,6 +22,8 @@
             (final: prev: {
               # Because tests are failing on darwin: https://github.com/juspay/services-flake/pull/115#issuecomment-1970467684
               pgadmin4 = prev.pgadmin4.overrideAttrs (_: { doInstallCheck = false; });
+              # grafana builds are failing on aarch64-darwin latest nixpkgs-unstable: https://github.com/NixOS/nixpkgs/issues/273998
+              grafana = inputs.nixpkgs-2023.legacyPackages.${system}.grafana;
```

Comment:
@shivaraj-bh , i have pinned grafana to an older commit since it fails to build after updating `flake.lock`.

is this a good idea?

Reply:
To avoid confusion, we usually keep only one nixpkgs input. I think we can overlay grafana, in `nixpkgs` to a version that works. Let me try it out.